client/src/components/Search/index.js

```diff
@@ -59,29 +58,27 @@ export default class Search extends Component {
    *
    * @param {*} placeDetail 
    */
+  
   onPlaceFound(placeDetail) {
```

Comment:
Also we should rename this function to "handleSelectedPlace" or some derivative of "handle..." since its handling the action of clicking on search prediction.